### ISSUE TITLE
provide binary to manage multiple executables in ~/.pandoc/bin

### ISSUE THREAD

**Issue by nichtich** [nichtich/Pandoc-Wrapper#29]
* List releases (via `Pandoc::Release->list`)
* Check new releases since newest available in `~/.pandoc/bin`
* Download new release(s)
* List executables and mark which is used by default
* Change symlink to select default version

**Comment by vanHoesel** [comment]
See also issue #35

**Comment by nichtich** [comment]
This could be used similar to [nvm](https://github.com/creationix/nvm), [perlbrew](https://perlbrew.pl/), [pyenv](https://github.com/pyenv/pyenv) etc. so best practice can be taken from these. How should the executable be named (e.g. `pdvm`)? Is it enough to switch executables or should template directory also be changed?

**Comment by vanHoesel** [comment]
it's certainly more safe to symlink directories too, so each install will have it's own environment. But i am a bit concerned... if this is for testing purposes (why else would one run against multiple versions of pandoc?), i think you'd be better off setting up testing hooks on Travis or things such like that, rather then trying to build something like a Perlbrew.

I'm aware one can use Perlbrew for running test agains different versions of Perl, but it's main purpose is to give users/devops the simple solution off installing a local version of Perl and managing local-lib etc.

Buy hey, i'm not that much into Pandoc as you are, and if you have good reasons to manage multiple Pandoc executable's, i am all fine with it.

**Comment by nichtich** [comment]
Yes, maybe this is overkill and documentation how to bootstrap (as [sketched here](https://github.com/nichtich/Pandoc-Wrapper/issues/35#issuecomment-398899638)) is enough (#13). I think this could also be useful for non-Perl programmers.

**Comment by nichtich** [comment]
This only need to be documented (how to switch pandoc versions, e.g. for testing templates and filters).

* install new version (`perl -MPandoc::Release -E 'latest->download->symlink'`)
* list available versions (`pandoc<tab>` if PATH is set)
* select version (`perl -MPandoc -E 'new Pandoc(2.1)->symlink'`)

It might be simplified, e.g. to

    perl -MPandoc::Release -e update
    perl -MPandoc::Release -e select(2.1)
